extensions/functions_arithmetic.yaml

```diff
@@ -469,6 +469,27 @@ scalar_functions:
       - args:
           - value: fp64
         return: fp64
+  -
+    name: "factorial"
+    description: >
+      Return the factorial of a given integer input.
+
+      The factorial of 0! is 1 by convention.
+
+      Negative inputs will raise an error.
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
```

Comment:
Should we specify the name for the options here?
`- name: overflow`

Reply:
yes!  nice catch.